cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,54 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SettingsPageObject from '../support/pages/settings.pageObject';
+
 describe('Settings page', () => {
-  before(() => {
+  const settingsPage = new SettingsPageObject();
 
+  before(() => {
+    cy.task('db:clear');
   });
 
   beforeEach(() => {
+const settingsPage = new SettingsPageObject();
 
+describe('Settings page', () => {
+    cy.task('db:clear');
   });
```

Comment:
There is a misplaced `describe` block inside the `beforeEach` hook. This is likely a mistake, as `describe` blocks should not be nested inside hooks. Remove this block to maintain the correct test structure.